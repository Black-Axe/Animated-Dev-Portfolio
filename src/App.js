import { BrowserRouter, Routes, Route , Navigate } from 'react-router-dom';
import HomePage from './pages/Home';
import './App.css';
import Blog from "./components/blog/Blog";
import BlogDetails from "./components/blog/BlogDetails";
import Error404 from './pages/Error404';


function App() {
  return (
    <BrowserRouter basename={'/'} element={<HomePage />}>
      <Routes>
        <Route path="/" element={<HomePage />} />
        <Route path="/articles" element={<Blog />} />
        <Route path="/articles/:blogSlug" element={<BlogDetails />} />
        <Route path="/404" element={<Error404 />} />
        <Route path="*" element={<Navigate to="/404" replace />} />
      </Routes>
    </BrowserRouter>
  );
}

export default App;
