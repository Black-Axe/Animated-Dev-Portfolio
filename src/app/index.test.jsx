import React from 'react'
import { expect, it } from 'vitest'
import App from './index'

it('renders snapshot of app', () => {
  expect(<App />).toMatchSnapshot()
})
