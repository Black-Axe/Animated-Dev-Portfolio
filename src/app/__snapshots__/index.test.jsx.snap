// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders snapshot of app 1`] = `<App />`;
